buildSrc/src/main/groovy/com/uber/okbuck/generator/RetroLambdaGenerator.groovy

```diff
@@ -22,7 +23,11 @@ final class RetroLambdaGenerator {
         String outputText = output.text
         outputText = outputText.replaceAll('gen-dir', target.rootProject.file(BuckConstants.DEFAULT_BUCK_OUT_GEN_PATH).absolutePath)
                 .replaceAll('retrolambda-jar', target.retroLambdaJar)
-        outputText = outputText.replaceAll('android-jar', target.bootClasspath)
+        if (target instanceof AndroidTarget) {
+            outputText = outputText.replaceAll('android-jar', target.bootClasspath ? target.bootClasspath : "")
```

Comment:
Nit pick but you can replace `target.bootClasspath ? target.bootClasspath : ""` with `target.bootClasspath ?: ""` I believe